bin/get_version_local.sh

```
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# Get the date, first 4 chars of branch name and short commit hash
+date=$(date -u +"%Y%m%d")
+branch=$(git rev-parse --abbrev-ref HEAD | cut -c1-4)
+commit=$(git rev-parse --short HEAD)
+clique=${1:-"."}
+
+uppercase_char() {
+    local char=$1
+
+    (echo $char | tr '[a-z]' '[A-Z]' 2> /dev/null) || echo "${char^^}"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)